packages/react-native/src/InAppMessaging/components/CarouselMessage/__tests__/CarouselMessage.spec.tsx

```diff
@@ -1,55 +1,92 @@
 import React, { ReactElement } from 'react';
-import { act, create, ReactTestRenderer } from 'react-test-renderer';
+import { render } from '@testing-library/react-native';
 
-import { Carousel } from '../../../../primitives';
+import { IN_APP_MESSAGING_TEST_ID } from '../../../constants';
 import CarouselMessage from '../CarouselMessage';
+import { defaultStyle } from '../styles';
+import { ReactTestRendererJSON } from 'react-test-renderer';
```

Comment:
If we are updating the unit tests to no longer use `react-test-renderer` directly than i would assume we would stop using the types as well?

Reply:
`react-native-testing-library` is built on top of react-test-renderer thus has a dependency on it, as noted in their [readme](https://github.com/callstack/react-native-testing-library#installation) as well.